Sources/ComposableArchitecture/Observation/Binding+Observation.swift

```diff
@@ -78,12 +78,81 @@
     }
   }
 
+  private final class BindableActionDebugger<Action>: Sendable {
```

Comment:
The work required to reuse the existing debugger seemed to not be worth it, especially if we're removing those APIs in 2.0.